src/odemis/gui/cont/tabs.py

```diff
@@ -3918,78 +3922,188 @@ def _onClickFocus(self, evt):
         """
         self.tab_data_model.autofocus_active.value = not self.tab_data_model.autofocus_active.value
 
+    def _createFocusStreams(self, focuser, hw_settings_config):
+        """
+        Initialize a stream to see the focus, and the slit for lens alignment.
+        :param focuser: the focuser to get the components for (currently ccd_focuser)
+        :param hw_settings_config: hw config for initializing BrightfieldStream
+        :return: all created focus streams
+        """
+        focus_streams = []
+        dets = GetSpectrometerFocusingDetectors(focuser)
+        # One "line" stream per detector
+        # Add a stream to see the focus, and the slit for lens alignment.
+        # As it is a BrightfieldStream, it will turn on the emitter when
+        # active.
+        for d in dets:
+            speclines = acqstream.BrightfieldStream(
+                "Spectrograph line ({name})".format(name=d.name),
+                d,
+                d.data,
+                emitter=None,  # actually, the brightlight, but the autofocus procedure takes care of it
+                focuser=focuser,
+                detvas=get_local_vas(d, hw_settings_config),
+                forcemd={model.MD_POS: (0, 0),
+                         model.MD_ROTATION: 0},
+                acq_type=model.MD_AT_SLIT,
+            )
+            speclines.tint.value = (0, 64, 255)  # colour it blue
```

Comment:
Maybe the imported blue should be a color defined in 

> src/odemis/gui/__init__.py

So maybe import from this place or reference to it in a comment

Reply:
Done as requested.